verification/v0/phoneVerification.yaml

```diff
@@ -69,13 +69,6 @@ paths:
         - metadata
       summary: Phone Verification API Health
       operationId: getPhoneVerificationHealth
-      parameters:
-        - name: apiKey
-          in: header
-          description: 'API authentication key for the customer account'
-          required: true
-          schema:
-            type: string

```

Comment:
The `apiKey` header is duplicated in the documentation. Once as a parameter and once as a security scheme. Am I right by removing this?